x-pack/plugins/viz_editor/public/frame/main/main.tsx

```diff
@@ -163,20 +178,22 @@ export function Main(props: MainProps) {
       )}
       <EuiPageBody className="vzBody">
         <DroppablePane
+          {...props}
           visModel={state.visModel}
           getAllSuggestionsForField={getAllSuggestionsForField}
           onChangeVisModel={onChangeVisModel}
         >
           <EuiPageContentBody>
             <EuiFlexGroup direction="column">
               <EuiFlexItem grow={5}>
-                {WorkspacePanel ? (
-                  <WorkspacePanel {...panelProps}>
+                {hasData &&
```

Comment:
Cleanup - only render if there is a query configured (prevents errors for empty expressions)